apps/sv/frontend/src/components/governance/ProposalVoteForm.tsx

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useForm } from '@tanstack/react-form';
+import { z } from 'zod';
+import { useSvAdminClient } from '../../contexts/SvAdminServiceContext';
+import { useMutation, UseMutationResult } from '@tanstack/react-query';
+import { isValidUrl } from '../../utils/validations';
+import { ContractId } from '@daml/types';
+import { VoteRequest } from '@daml.js/splice-dso-governance/lib/Splice/DsoRules';
+import { ProposalVote, VoteStatus } from '../../utils/types';
+import {
+  Alert,
+  Box,
+  Button,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  TextField,
+  Typography,
+} from '@mui/material';
+interface CastVoteArgs {
+  accepted: boolean;
+  url: string;
+  reason: string;
+}
+
+interface ProposalVoteFormProps {
+  voteRequestContractId: ContractId<VoteRequest>;
+  currentSvPartyId: string;
+  votes: ProposalVote[];
+}
+
+export const ProposalVoteForm: React.FC<ProposalVoteFormProps> = props => {
```

Comment:
FYI this form looks different to what's on figma. I had a chat with UX and we agreed on this new design but there was no need to mock it :)